### PR TITLE
update user guide

### PR DESCRIPTION
**Pull request by KohlerHECTOR**
## Description
PR for issues #325 #353 
I remove some place holders in the user guide that we would probably never do and I did a user guide page for plotting and statitstics.

## Checklist
- \[ ] My code follows the style guideline
To check :
   black --check examples rlberry *py
   flake8 --select F401,F405,D410,D411,D412 --exclude=rlberry/check_packages.py --per-file-ignores="__init__.py:F401",
- \[ ] I have commented my code, particularly in hard-to-understand areas,
- \[ ] I have made corresponding changes to the documentation,
- \[ ] I have added tests that prove my fix is effective or that my feature works,
- \[ ] New and existing unit tests pass locally with my changes,
- \[ ] If updated the changelog if necessary,
- \[ ] I have set the label "ready for review" and the checks are all green.

### CODE REVIEW

File: docs/basics/userguide/plot_stats.md [hunk 1]
```diff
@@ -0,0 +1,7999 @@
...
+    seed=42,
+    fit_budget=int(1e4),
+    n_fit=10,
+    agent_name="Sb3-PPO",
+)
+first_agent.fit()
+second_agent.fit()
+```
+
+    [38;21m[INFO] 14:38: Running ExperimentManager fit() for Sb3-A2C with n_fit = 10 and max_workers = None. [0m
```
Comment (TimotheeMathieu) [inline docs/basics/userguide/plot_stats.md:45]: Please clean this up. I don't think we need the outputs of experiments in the doc in this way. It can be interesting once to show what to expect but this is clearly too much output.

File: docs/basics/userguide/plot_stats.md [hunk 2]
```diff
@@ -0,0 +1,7999 @@
...
+    [38;21m[INFO] 14:46: [Sb3-PPO[worker: 9]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 45.69 | rollout/ep_len_mean = 45.69 | time/fps = 117 | time/time_elapsed = 69 | time/total_timesteps = 8192 | train/learning_rate = 0.0003 | train/entropy_loss = -0.6443713787943125 | train/policy_gradient_loss = -0.012846545978391077 | train/value_loss = 55.9754634976387 | train/approx_kl = 0.008374381810426712 | train/clip_fraction = 0.0560546875 | train/loss = 18.558977127075195 | train/explained_variance = 0.21047407388687134 | train/n_updates = 30 | train/clip_range = 0.2 |  [0m
+    INFO:rlberry_logger:[Sb3-PPO[worker: 9]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 45.69 | rollout/ep_len_mean = 45.69 | time/fps = 117 | time/time_elapsed = 69 | time/total_timesteps = 8192 |
```
Comment (TimotheeMathieu) [inline docs/basics/userguide/plot_stats.md:1405]: Same. No need for this output.

Comment (TimotheeMathieu) [inline docs/basics/userguide/plot_stats.md:1123]: Please do not include colab stuffs. They are not useful for the doc and may cause bugs.

Comment (TimotheeMathieu) [inline docs/basics/userguide/plot_stats.md:7999]: Please, write an explanation on how to interpret the results.

File: docs/user_guide.md [hunk 3]
```diff
@@ -25,24 +25,10 @@ It's the repository used for teaching purposes. These are mainly basic agents an
...
 - [Seeding](seeding_page)
 - [Save & Load Experiment](save_load_page)
-- [Save & Load Agents (In construction)]()
-- [Save & Load Data (In construction)]()
-## Advanced Usage
-- [Custom Agents (In construction)]()
-- [Custom Environments (In construction)]()
-- [Using extrenal libraries](external) (like [Stable Baselines](stable_baselines) and [Gymnasium](Gymnasium_ancor))
-- [Transfer Learning (In construction)]()
-- [AdaStop(In construction)]()
```
Comment (TimotheeMathieu) [inline docs/user_guide.md:48]: Could you re-add the links so that we have an idea of where we are going.

**Review by TimotheeMathieu — COMMENTED** [review]
Thanks for this addition. However, for now, it is a bit difficult to review this. Can you clean up all the outputs?

File: docs/basics/userguide/plot_stats.md (hunk 2 above)
Comment (KohlerHECTOR) [reply docs/basics/userguide/plot_stats.md:7999]: Done

Comment (KohlerHECTOR) [reply docs/basics/userguide/plot_stats.md:1123]: Don't know how to remove, this was generated automatically with a jupyter to markdown converter.

File: docs/basics/userguide/plot_stats.md (hunk 1 above)
Comment (KohlerHECTOR) [reply docs/basics/userguide/plot_stats.md:45]: removed all unnecessary outputs.

File: docs/user_guide.md (hunk 3 above)
Comment (KohlerHECTOR) [reply docs/user_guide.md]: re added links.

File: docs/basics/userguide/plot_stats.md (hunk 2 above)
Comment (TimotheeMathieu) [reply docs/basics/userguide/plot_stats.md:69]: You can just remove them manually.

**Comment by TimotheeMathieu** [comment]
A part has been integrated into visualization user guide, another part is now in the new repo with the avec tutorial template. Closing.
